docs/flexbox.md

```diff
@@ -1197,6 +1197,249 @@ const styles = StyleSheet.create({
 export default App;
 ```
 
+## Gap, Row Gap and Column Gap
+
+- [`rowGap`](layout-props#rowGap) property sets the size of the gap (gutter) between an element's rows.
+
+```SnackPlayer name=Row%20Gap
+import React, { useState } from "react";
+import { View, TouchableOpacity, Text, StyleSheet, TextInput } from "react-native";
+
+const RowGap = () => {
+  const [rowGap, setRowGap] = useState(10);
+
+  return (
+    <PreviewLayout
+      label="rowGap"
+      handleRowGapChange={setRowGap}
+      rowGap={rowGap}>
+      <View
+        style={[styles.box, { backgroundColor: "orangered" }]}
+      />
+      <View
+        style={[styles.box, { backgroundColor: "orange" }]}
+      />
+    </PreviewLayout>
+  );
+};
+
+const PreviewLayout = ({
+  label,
+  children,
+  values,
+  selectedValue,
+  setSelectedValue,
+  handleRowGapChange,
+  rowGap
+}) => (
+  <View style={{ padding: 10, flex: 1 }}>
+    <Text style={styles.label}>{label}</Text>
+    <View style={{rowGap: 4}}>
+      <Text>Edit Row Gap</Text>
+      <TextInput style={styles.input} value={rowGap} onChangeText={(v) => handleRowGapChange(Number(v)) } />
```

Comment:
This line is 103 characters long and seems to overflow the Snack player example. Could we make sure to run the samples through Prettier?

<img width="545" alt="image" src="https://user-images.githubusercontent.com/835219/200094147-c592fd85-7dcf-48e0-8f6c-f09684fb21eb.png">